### PR TITLE
[Core] Minor clean up PointerVectorSet

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**
Minor clean up PointerVectorSet

**🆕 Changelog**
- Minor clean up PointerVectorSet

### CODE REVIEW

File: kratos/containers/pointer_vector_set.h [hunk 1]
```diff
@@ -448,8 +386,8 @@ class PointerVectorSet final
 
     iterator erase(iterator pos)
     {
-		if (pos.base() == mData.end())
-			return mData.end();
+	if (pos.base() == mData.end())
+	    return mData.end();
         iterator new_end = iterator( mData.erase( pos.base() ) );
         mSortedPartSize = mData.size();
```
Comment (EduardGomezEscandell) [inline kratos/containers/pointer_vector_set.h:392]:
```suggestion
	    if (pos.base() == mData.end())
	        return mData.end();
        iterator new_end = iterator( mData.erase( pos.base() ) );
        mSortedPartSize = mData.size();
```
Misleading indentation.

Comment (EduardGomezEscandell) [inline kratos/containers/pointer_vector_set.h:392]:
```suggestion
        if (pos.base() == mData.end())
            return mData.end();
        iterator new_end = iterator( mData.erase( pos.base() ) );
        mSortedPartSize = mData.size();
```
tabs->spaces

Comment (loumalouomega) [reply kratos/containers/pointer_vector_set.h:392]: Corrected, but doesn' appear

**Comment by loumalouomega** [comment]
Ping @KratosMultiphysics/technical-committee

**Comment by loumalouomega** [comment]
Ping @rubenzorrilla

**Review by rubenzorrilla — APPROVED** [review]
Only aesthetics and clean up. Thanks @loumalouomega

**Pull request merged**